staging/src/k8s.io/apimachinery/pkg/watch/mux.go

```diff
@@ -127,18 +129,20 @@ func (m *Broadcaster) Watch() Interface {
 		}
 		m.watchers[id] = w
 	})
+	if w == nil {
```

Comment:
Add a comment saying this matches previous behavior, but that we're open to reassessing in the future.

Reply:
done